bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/ide/dialogs/IDEWorkspacePreferencePage.java

```diff
@@ -197,6 +200,25 @@ private void createMissingNaturePref(Composite parent) {
 		missingNatureSeverityCombo.load();
 	}
 
+	/**
+	 * @param comboParent
```

Comment:
Please remove comment.

Reply:
removed that jdoc with (even wrong) parameter name. Thanks for finding.

(I had added this for consistency with the other methods - but I agree that the comment doesn't add any value. Happy to drop it)